source/tutorials/importing.rst

```diff
@@ -84,12 +85,14 @@ Format description
 
 Paired-end "`Earth Microbiome Project (EMP) protocol`_" formatted reads should have three ``fastq.gz`` files total:
 
-1. one ``fastq.gz`` file that contains the forward sequence reads,
-2. one ``fastq.gz`` file that contains the reverse sequence reads,
+1. one ``forward.fastq.gz`` file that contains the forward sequence reads,
+2. one ``reverse.fastq.gz`` file that contains the reverse sequence reads,
 3. and a third that contains the associated barcode reads
```

Comment:
Since we are outlining the other two filenames, probably should explicitly list this one, too:

```suggestion
3. one ``barcodes.fastq.gz`` file that contains the associated barcode reads
```